homeassistant/components/azure_data_explorer/client.py

```diff
@@ -0,0 +1,96 @@
+"""Setting up the ingest client."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+import io
+import logging
+
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+from azure.kusto.data.data_format import DataFormat
+from azure.kusto.ingest import (
+    IngestionProperties,
+    ManagedStreamingIngestClient,
+    QueuedIngestClient,
+    StreamDescriptor,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
```

Comment:
If it's a dataclass then you shouldn't need a init method!

Reply:
removed @dataclass